apps/system/js/bluetooth_transfer.js

```diff
@@ -643,6 +645,33 @@ var BluetoothTransfer = {
     var body = {id: 'unknownMediaTypeToOpenFile', args: {fileName: fileName}};
     CustomDialog.show('cannotOpenFile', body, confirm, null, screen)
     .setAttribute('data-z-index-level', 'system-dialog');
-  }
+  },
 
+  _cleanupTempFileInDir: function bt__cleanupTempFileInDir(evt) {
+    // Try to get the file from bluetooth temp folder.
+    var filePath = '.bluetooth/tmp/' + evt.fileName;
+    var storageType = 'sdcard';
+    var storage = navigator.getDeviceStorage(storageType);
```

Comment:
Yes, it's a potential issue there. We don't keep the instance of device storage because the instance is possible out of useful in case of changed default media storage. I hit the problem before.
Bug 1110068 - [Settings]DUT can't receve file via bluetooth after you switch media location to internal.
Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1110068

Bug 1115496 - [DeviceStorage] Provide event from outdated DeviceStorage instance object to notify Gaia inconsistently.
https://bugzilla.mozilla.org/show_bug.cgi?id=1115496

We have to considerate all of the use case since default media storage is possible to be changed any time. Thanks for reminder here.